api/src/backend/api/apps.py

```diff
@@ -1,4 +1,27 @@
+import logging
+import os
+
+from pathlib import Path
+
 from django.apps import AppConfig
+from django.conf import settings
+
+from config.custom_logging import BackendLogger
+from config.env import env
+
+logger = logging.getLogger(BackendLogger.API)
+
+SIGNING_KEY_ENV = "DJANGO_TOKEN_SIGNING_KEY"
+VERIFYING_KEY_ENV = "DJANGO_TOKEN_VERIFYING_KEY"
+
+SIGNING_KEY_FILE = "jwt_signing_key.pem"
+VERIFYING_KEY_FILE = "jwt_verifying_key.pem"
+
+KEYS_DIRECTORY = (
+    Path.home() / ".keys" / "prowler-api"
```

Comment:
I know we talked about it but now I'm wondering if this is redundant as the container is only for the API.

Reply:
Do you mean from sources?